### PR TITLE
A stupid lazy fix but a needed fix. 

### PR DESCRIPTION
**Pull request by Summoners**
[Fixes SVN issue 1217... again.]

Instead of coding special snowflake code just for the purpose of *hiss for aliens, it was decided to do the change that wouldn't change anything but still work when you typed in *hiss.

What this fixes:

Instead of _hiss coming up as "_hi is a invalid command" it now works due to the shortening the requested emote s to *his.

[Related: I've had to shorten *hiss to *his before on 2013-02-26 back in the SVN days. I guess I come out of the woodwork to fix *hiss on aliens every couple years.]

### CODE REVIEW

**Comment by WJohn** [comment]
So *his will make the hissing sound? I'm assuming that you can't use *hiss because of saycode nonsense having already taken it?

**Comment by Summoners** [comment]
@WJohn This doesn't add the *hiss sound back into the emotes, this simply allows you to do the *hiss emote. The issue comes from the removal of the extra letter s. This is made for the purpose of doing a emote like *dances and instead it does the *dance emote. However the issue happens when the player types *hiss and it decides to shorten it to *his (then to *hi).

This simply makes it accept *his as a emote so when a player types *hiss and it removes the s it still works.

**Comment by RemieRichards** [comment]
According to the code, *_hiss should work.

**Comment by Summoners** [comment]
*_hiss does not work, you have to type *_hisssss to get *_hiss but thats not a command.
(Unusable emote '_hiss'. Say *help for a list.)

That underscore emote code thing is a lie that makes it remove an extra -2 s's. It doesn't even activate ANY emotes with _. 

I might go into the code and purge every single "_" in a future patch

**Comment by RemieRichards** [comment]
How about, remove the code that removes s's, and duplicate the functionality of "dance" and "dances" by having the switch case for if("dance") be if('dance","dances") and the same for all others that violate it.

Why ruin other emotes because one emote happens to want to work with an s, but decided not to include the case.

tl;dr:
```if("dance")``` --> ```if("dance","dances")```
kill off -s code.
do for all emote() procs (about 6-7)

**Comment by Summoners** [comment]
@RemieRichards Dances is an example but if you want I guess I could do that to every emote in the game.

**Comment by RemieRichards** [comment]
@Summoners it's a more logical solution, and causes less problems in the future, so I'd appreciate it.
if an emote seems too illogical to have an -s variant, then skip that case (although I can't think of any)

**Comment by Summoners** [comment]
@RemieRichards Alrighty then, I'll get to it.

**Comment by Summoners** [comment]
There we go, just gave almost every emote a plural + their normal... except cry has "cry, crys, cries" because I know the playerbase.

Oh and I changed an alien emote to *screech instead of *scretches

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -8,25 +8,22 @@
 
 	var/param = null
 
-	if (findtext(act, "-", 1, null))
+	if (findtext(act, "-", 1, null)) //Removes dashes for npcs "EMOTE-PLAYERNAME" or something like that, I ain't no AI coder. It's not for players. -Sum99
```
Comment (RemieRichards) [inline code/modules/mob/living/emote.dm]: It's actually have you have an emote with arguments.

*point-Lindsay Donk
for example

**Comment by RemieRichards** [comment]
@Summoners Great work.
This could do with a changelog entry, informing people that emotes are a bit more user friendly now.
Perhaps also specifying the fix of *hiss to give people a basis for what's changed.

:+1:

File: code/modules/mob/living/emote.dm (hunk 1 above)
Comment (Summoners) [inline code/modules/mob/living/emote.dm]: @RemieRichards Ah, never seen anyone ever use it.

Comment (RemieRichards) [inline code/modules/mob/living/emote.dm]: Neither have I, hahaha

**Comment by Summoners** [comment]
@RemieRichards Changelog added.
